File: scripts/migrate.js
import { execSync } from "child_process";

try {
  console.log("Pushing Prisma schema to database...");
  execSync("npx prisma db push --skip-generate", { stdio: "inherit" });
  console.log("Database schema pushed successfully!");
} catch (error) {
  console.error("Failed to push schema:", error.message);
  process.exit(1);
}

File: scripts/migrate.py
#!/usr/bin/env python3
import subprocess
import sys
import os

os.chdir('/vercel/share/v0-project')

try:
    print("[v0] Pushing Prisma schema to database...")
    result = subprocess.run(
        ["npx", "prisma", "db", "push", "--skip-generate"],
        capture_output=True,
        text=True
    )
    
    print(result.stdout)
    
    if result.returncode != 0:
        print("Error:", result.stderr)
        sys.exit(1)
    
    print("[v0] Database schema pushed successfully!")
except Exception as e:
    print(f"[v0] Error: {str(e)}")
    sys.exit(1)
